sdk/samples/iot/README.md

```diff
@@ -277,7 +283,7 @@ The following samples use SAS authentication to connect to Azure IoT Hub or Azur
 
```

Comment:
Section starting at line 273 is only for CMD.  Needs section for PowerShell.

Reply:
Updated later